service/test/integration2_test.go

```diff
@@ -1387,3 +1387,28 @@ func TestClientServer_collectBreakpointInfoError(t *testing.T) {
 		assertNoError(state.Err, t, "Continue()")
 	})
 }
+
+func TestClientServer_PR1055(t *testing.T) {
+	protest.AllowRecording(t)
```

Comment:
There should be a comment here describing what this is actually testing, as a future reminder, something like "Next should behave like continue when the program exits and return the error in state.Err".

Other than that looks good.

Reply:
I've updated the PR accordingly. Thank you for your help!